src/tools/spake2p/Cmd_GenVerifier.cpp

```diff
@@ -308,16 +312,17 @@ bool Cmd_GenVerifier(int argc, char * argv[])
     if (gOutFileName == nullptr)
     {
         fprintf(stderr, "Please specify the output file name, or - for stdout.\n");
-        return false;
+        ExitNow(ret = false);
```

Comment:
So why are we doing these "exit" things before the part where we really have to worry bout `outFile`?

This would be clearer if the decl of `outFile` were moved to right before the block that sets it.  Anything above that decl can return, not exit.

Reply:
I've change implementation as @andy31415 suggest. Now I'm using C++ methods and there is no more `exit` things